cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -153,8 +153,17 @@
           "default": false
         },
         "sudo": {
-          "type": ["boolean", "string"],
-          "description": "Sudo rule to use or false. Absence of a sudo value or ``false`` will result in no sudo rules added for this user. DEPRECATED: the value ``false`` will be deprecated in the future release. Use ``null`` or no ``sudo`` key instead."
+          "oneOf": [
+            {
+              "type": ["string", "null"],
+              "description": "Sudo rule to use or false. Absence of a sudo value or ``null`` will result in no sudo rules added for this user."
+            },
+            {
+              "type": "boolean",
+              "deprecated": true,
+              "description": "The value ``false`` will be deprecated in the future release. Use ``null`` or no ``sudo`` key instead."
```

Comment:
```suggestion
              "description": "The value ``false`` will be dropped after April 2027. Use ``null`` or no ``sudo`` key instead."
```

Reply:
done in c60e194b931ac88abd48c8887f6e77c0db185a64, thanks!